agent/agent-profiler/agent-service-profiler/src/main/java/com/microsoft/applicationinsights/serviceprofilerapi/profiler/JfrProfiler.java

```diff
@@ -114,30 +114,23 @@ public JfrProfiler(ServiceProfilerServiceConfig configuration) {
    *     enabled.
    */
   @Override
-  public boolean initialize(
+  public void initialize(
```

Comment:
can we use initialise ? it seems it's pretty consistent everywhere else.

Reply:
initialise is @johnoliver's british spelling 😂